src/index.ts

```diff
@@ -331,10 +331,11 @@ function initialSetup() {
 			canBeFlashed (bool)
 			supportsChat (bool)
 			*/
-			obj = JSON.parse(String(obj)); //Re-parse JSON
 
 			logger(`Received JSON object: ${obj}`); //To make it easier for devs to verify the JSON objects they're sending
 
+			obj = JSON.parse(String(obj)); //Re-parse JSON

```

Comment:
I would change this to 

`if(typeof obj !== 'object' && obj !== null) {
     logger(`Received JSON object: ${obj}`, 'info-quiet');
     obj = JSON.parse(String(obj)); //Re-parse JSON
}`

That way it does not parse an already parsed JSON object